microcosm_pubsub/dispatcher.py

```diff
@@ -34,21 +31,14 @@ def handle_batch(self, bound_handlers):
 
         :returns: a `DispatchResult` with metrics around message consumption
```

Comment:
Nit: doc now out of date.

Reply:
Also, if we're embracing 3.7+ (I see direct dataclass import), might I suggest adding typehinting?